SetupDataPkg/SetupDataDxe/SchemaXmlHashDriver/SchemaXmlHash.c

```diff
@@ -0,0 +1,54 @@
+/** @file SchemaXmlHash.c
+  Set config xml hash to BIOS variable
+
+  Copyright (c) Microsoft Corporation.
+
+**/
+#include <Base.h>
+#include <Library/DebugLib.h>
+#include <Library/BaseLib.h>
+#include <Library/UefiRuntimeServicesTableLib.h>
+#include <ConfigStdStructDefs.h>
+#include <Generated/ConfigClientGenerated.h>
+#include <Generated/ConfigDataGenerated.h>
+#include <Generated/ConfigProfilesGenerated.h>
+
+/**
+  Module entry point that register to set PlatformConfigData.xml hash variable
+
+  @param FileHandle                     The image handle.
+  @param PeiServices                    The PEI services table.
+
+  @retval Status                        From internal routine or boot object, should not fail
+**/
+EFI_STATUS
+EFIAPI
+SchemaXmlHashDxeEntry (
+  IN EFI_HANDLE        ImageHandle,
+  IN EFI_SYSTEM_TABLE  *SystemTable
+  )
+{
+  EFI_STATUS  Status;
+  EFI_GUID schema_xml_hash_guid = SCHEMA_XML_HASH_GUID;
+
+  DEBUG((DEBUG_ERROR, "In %a\n", __FUNCTION__));

```

Comment:
This and all entry/exit prints should be removed

Reply:
done
https://github.com/microsoft/mu_feature_config/commit/05d3f945944ae7b5c161faaba1792f9cc9d54c6c#diff-7dba4a44ef45cf41c4472c5d3c349c297d09d2c5237c7fe2cb525c0697b93edaL34